src/main/resources/changelogs/0004-add-admin.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="1_add_admin_role" author="raluca">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0"><![CDATA[
+                select COUNT(*) from role where role_name='ADMIN';
+            ]]></sqlCheck>
+        </preConditions>
+        <sql>
+            insert into role(id,role_name) values(79,'ADMIN');
```

Comment:
don't insert the db id, let mysql populate the column

Reply:
done